test/jdk/sun/security/pkcs/pkcs9/PKCS9AttrTypeTests.java

```diff
@@ -166,6 +170,11 @@ public static void main(String[] args) throws Exception {
                 // a DerValue object to be consumed by PKCS9Attribute.
                 PKCS9Attribute p9Attr = new PKCS9Attribute(
                         new DerValue(Utils.toByteArray(entry.getValue())));
+
+                // Encoding is supported
+                DerOutputStream dos = new DerOutputStream();
+                p9Attr.encode(dos);
```

Comment:
Should we check the encoding has the expected value? Otherwise, it looks like we only requires that no exception is thrown?

Reply:
I see, not surprising, perhaps just checking that it's not empty? Or even that may not hold true?